fix_includes.py

```diff
@@ -2169,6 +2173,13 @@ def main(argv):
   parser.add_option('--nosafe_headers', action='store_false',
                     dest='safe_headers')
 
+  parser.add_option('--reorder', default=False, action='store_true', dest='reorder',
```

Comment:
If you use `action='store_true'` you don't need the default as the parser already knows it

Reply:
I don't have a strong preference for "on by default" or "requires a flag" for this feature.  Some people were of the opinion this should be the default.  I'll agree with whatever the maintainer(s) want.

Reply:
In the comment below you indicated the option will have no effect.  From my testing this is the behavior I see:

```
$ fix_includes.py -n -s /tmp/blah.cc | md5sum
513f864111d7efd9f66540eb373bebfc  -
$ fix_includes.py -n -s /tmp/blah.cc --no-reorder | md5sum
513f864111d7efd9f66540eb373bebfc  -
$ fix_includes.py -n -s /tmp/blah.cc --reorder --no-reorder | md5sum
513f864111d7efd9f66540eb373bebfc  -
$ fix_includes.py -n -s /tmp/blah.cc --reorder | md5sum
48316666f509f3d37221fb5b118a3574  -
```

Things to note from that test:

* `--no-reorder` doesn't change default behavior
* `--no-reorder` inverts the behavior of `--reorder`
* `--reorder` produces different output from the default

In my initial implementation I wasn't specifying `default`; in that scenario the default was based on the order arguments were declared.  I'm sure you'd agree that isn't ideal.

Unless I'm missing something I see only 2 worthwhile choices (unless someone devises a better answer):

* Maintainer(s) decide what the default should be, then eliminate the flag that corresponds with the default
* Keep it the way it is

Reply:
Thanks for the explanation, I see what you mean now. Suggestion from here: Remove the default from the one not matching the default. From:
>In my initial implementation I wasn't specifying default; in that scenario the default was based on the order arguments were declared. I'm sure you'd agree that isn't ideal.

This is not fully true. You used `default=False, action='store_false'` here (on purpose as I see now). Omitting the default clause would implicitly change that to `default=True, action='store_false'`. Hence your observation about order. But removing the default from `default=False, action='store_true'` will not change anything.

As what you are doing here is "strange", a comment in the code would be good to highlight, that you have 2 switches for the same option and `default` is used on purpose.

Reply:
Err, no! As I explained above removing the `default=False` from the first line does not change anything as that is the default for `store_true`. Removing it from the 2nd does change the meaning as it is **not** the default for `store_false` which makes that required.